.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -3,21 +3,20 @@ description: Create a report to help us improve
 title: "[Bug]: "
 labels: ["bug", "support_request"]
 assignees:
-  - octocat
+  - ilya-lavrenov
```

Comment:
can it be unassigned?
I don't want to be assigned to all bugs

Reply:
I have removed the assignees